djautotask/api.py

```diff
@@ -73,26 +73,31 @@ def retry_if_api_error(exception):
 
 
 def get_cached_url(cache_key):
-    return cache.get(cache_key)
+    return cache.get('Zone{}'.format(cache_key.capitalize()))
```

Comment:
Why the call to capitalize? Why not just be consistent when you use the key in the code? Its a small change though, I'll just make it and merge the issue.